### DISCUSSION TITLE
ability to create dev/test deployments for a feature branch

### DISCUSSION THREAD

**Discussion by andrewm4894** [LAION-AI/Open-Assistant#3433] [Ideas]
it would be very useful to be able to manually, or maybe via a GH action that can only be manually triggered, deploy a feature branch version of the app. 

So for example to be easily able to get feedback or have people test a feature branch or PR some way to just deploy it to some dev env like:

https://web.pr-123545.open-assistant.io
https://web.pr-some-branch-name.open-assistant.io

not sure if this will just be too complex of prohibitively expensive but could be handy if it was possible.

**Comment by yk** [comment]
probably doable by instrumenting the ansible playbook a bit
